client/landing/jetpack-cloud/sections/backups/components/activity-log/style.scss

```diff
@@ -0,0 +1,62 @@
+.activity-log {
+	// todo: there is some strangeness with the jetpack cloud css, these are temporary fixes
+	.filterbar__wrap.card {
```

Comment:
as per the comments these rules get `ActivityLog` looking presentable but are a hack for figuring out what the heck is actually happening with some of these css rules